net/apfree-wifidog/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2006,2015 OpenWrt.org

```

Comment:
No, either drop this, or use your own. (unless you actually have copyright assignment on file with openwrt)

Reply:
ok, I'll modify these copyright information